sig-testing/charter.md

```diff
@@ -0,0 +1,82 @@
+# SIG Testing Charter
+
+This charter adheres to the conventions described in the 
+[Kubernetes Charter README] and uses the Roles and Organization Management
+outlined in [sig-governance].
+
+## Scope
+
+SIG Testing is interested in effective testing of Kubernetes.  We do not write or
+troubleshoot the project's tests, but instead focus on tooling that makes it
+easier for the community to write and run tests, and to contribute, analyze and
+act upon test results.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Project CI and merge automation via tools such as [prow] and [tide]
+- Extraction of test results from GCS and populating a public accessible
+  BigQuery dataset via [kettle]
+- Display and analysis of test artifacts via tools like [gubernator], 
+  [testgrid], [triage] and [velodrome]
+- Configuration management of jobs and ensuring they use a consistent
+  process via tools such as [job configs], [kubetest]
+- Tools that facilitate local testing of kubernetes such as [greenhouse] 
+  and [kind]
+- Jobs that automate away project toil via [@fejta-bot]
+- Ensuring all of the above is kept running on a best effort basis
+
+#### Cross-cutting and Externally Facing Processes
+
+- The Release Team [test-infra role] is staffed by a memer of SIG Testing, as
+  such their responsibilities are within the scope of this SIG, including
+  the maintenance of release jobs
+- When rolling out changes that may potentially impact the project as a whole
+  we consult with SIG Contributor Experience, and follow [lazy consensus] by 
+  notifying kubernetes-dev, providing a deadline,and a rationale for the 
+  deadline if necessary.
+- We reserve the right to halt automation and infrastructure that we own,
+  or disable tests that we don't own if the project as a whole is being 
+  impacted
+- We are actively assisting with the transition of project infrastructure to
+  the CNCF and enabling non-Googlers to support this
+
+### Out of scope
+
+- We are not resonpsible for troubleshooting or writing tests or jobs for 
+  features owned by other SIGs
+- We are not responsible for ongoing maintenance of kubernetes' e2e test 
```

Comment:
For discussion: maybe this falls under the testing-commons subproject? But in terms of staffing I don't see ongoing maintenance

Reply:
discussed during today's meeting, we'll call it out as owned by this sig, but unstaffed... to paraphrase the meeting, we know where at least some of the bodies are buried, but that doesn't mean there were no more bodies buried on top